Automattic/sensei
# [Proposal] Use `senseiAssets` alias for relative assets in SCSS files

### Changes proposed in this Pull Request

* When importing Sensei SCSS assets into Sensei Pro, I'm running into issues where sometimes `sass-loader` isn't resolving relative paths correctly. I'm proposing we switch to using a named alias in the URLs that we can name in both Sensei and Sensei Pro's webpack.
* If we merge, I'll add this alias to our Sensei Pro webpack.

* I'd be open to just using a new `sensei` alias (to match what we have in Sensei Pro (then doing `sensei/assets`). I was quasi-wondering about conflicts, but that might not be an issue.

### Testing instructions
- Run `npm run build`
- Ensure it builds without errors.
- Check Sensei for missing styles.
